docs/usage_and_configuration/the_basics.md

```diff
@@ -4,11 +4,11 @@ Foundational knowledge on using and configuring Black.
 
 _Black_ is a well-behaved Unix-style command-line tool:
 
-- it does nothing if no sources are passed to it;
+- it does nothing if it finds no sources to format;
```

Comment:
New with this PR. With previous discussions about passing no sources I think it should be considered user error. Finding no files to format still passes and returns success.